### PR TITLE
Add SnailSVN-Helper

### PR DESCRIPTION
**Pull request by newnight**
- [X] I'm the package's author and/or maintainer.
- [X] I have have read [the docs][1].
- [X] I have tagged a release with a [semver][2] version number.
- [X] My package repo has a description and a README describing what it's for and how to use it.
- [ ] My package doesn't add context menu entries. *
- [X] My package doesn't add key bindings. **
- [ ] Any commands are available via the command palette.
- [ ] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [ ] If my package is a syntax it doesn't also add a color scheme. ***
- [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is SnailSVN helper 

There are no packages like it in Package Control.

My package is similar to ... However it should still be added because ...

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
What does your package do exactly? I don't really understand it and the Readme doesn't help. But I don't know what SnailSVN does so maybe it's just me.

Why is there a context menu? That makes more sense for interacting with selections etc., not for entire files. A menu item in the File menu would make more sense. And in any case all commands should be available via the command palette.

File: repository/s.json [hunk 1]
```diff
@@ -2565,6 +2565,19 @@
 				}
 			]
 		},
+		{
+			"name": "SnailSVN_Helper",
+			"description": "This is a SnailSVN Helper for Sublime",
```
Comment (braver) [inline repository/s.json:2570]: That much is clear from the name, but what does it do?

Comment (braver) [reply repository/s.json:2570]: Also, you can remove this and instead simply fill in a description on your GitHub repo. That will be used instead.

**Comment by braver** [comment]
Please respond to the feedback to continue the review.

**Comment by newnight** [comment]
SnailSVN is a TortoiseSVN-like Apache Subversion (SVN) client for Mac OS X, implemented as a Finder extension. 

official website is: https://langui.net/snailsvn/

this package will allow users to directly perform SVN operations in Sublime

**Comment by braver** [comment]
Alright, thanks. I have made several other points in my earlier feedback, please address those as well.

**Pull request closed without merging**